### DISCUSSION TITLE
Move package requests from Issues to Discussons

### DISCUSSION THREAD

**Discussion by hholst80** [void-linux/void-packages#44913] [General]
What do you think? I think it would significantly lower the number of issues and avoid the mixup of active and eternal backlog things.  Issues should be for more urgent issues with existing packages.  With over 700 issues, it isn't easy to navigate and help.

**Comment by classabbyamp** [comment]
already been discussed multiple times: #43081 #36700

**Comment by hholst80** [comment]
Seems the latest discussion was in favor of moving Package requests, yet nothing happened. Lack of steering/time? Sometimes people are too busy to realize they miss the big picture. 

I still think for most people such a division of concern would be beneficial. Almost nobody (except perhaps @Duncaen) can navigate 700 issues to find good tasks to work on.

A new package request is found by searching for it and upvoting. Different workflow from browsing issues to fix. Nobody goes around searching for random packages to include. We do this via process: we pick packages that have a high interest in the community.

**Reply by Duncaen** [reply]
You can simply ignore package requests with `-label:request` as search query.

**Reply by Duncaen** [reply]
> We do this via process: we pick packages that have a high interest in the community.

No we don't and we want maintainers that use packages and not just package random things because they are highly upvoted.
 Upvotes should have no say on what is being merged or prioritized, since its not what we do.

**Comment by hholst80** [comment]
> You can simply ignore package requests with `-label:request` as search query.

IMHO, there is likely something wrong when the number of issues closes up to a thousand in a single repo.  To me, it is a huge red flag for any software project that it has spiraled out of control or is just simply not maintained properly. [1]

Let's back up a step. Void packages are not included due to popularity but rather added on the basis that someone - a package maintainer - adds the package. So what purpose does a request Issue serve if it is not associated (or replaced by) a PR that actually adds the package?  What purpose does it fill? To discuss? To collect a popularity rating? Why are users adding these requests and to what benefit does Void allow them?

[1]: I will not debate this here, it is subjective.

**Reply by Duncaen** [reply]
The requests are just a collection of information, inspiration and maybe a mutex to stop contributors from working on the same package.

Some contributors might use package requests as a source of inspiration and package software they find interesting out of kindness even if they don't plan to use it. But the popularity by default means nothing. There are no employees that have to work on open issues, we are all just people contributing whatever they want and can in their free time.

**Reply by Duncaen** [reply]
Also closed or rejected requests will have a discussion and a reason attached, they can be linked to whenever someone asks for or looks up a package.

Open requests might have discussions about issues with packaging. There are a few packages that multiple people attempted to package, where the issue provides a central point for discussion and links to the "failed" pull requests.

I don't see how a forum like experience would help with that and that's why I'm against moving them into discussions.

I'm also not seeing a good reason to not use Issues for them, they work quite well, are integrated into the github experience and don't require yet another platform or workflow. The issues are well labeled and easily filtered. Its also not like we are doing anything weird here, NixOs is also using github issues as their primary way to track package requests and they have over 5k open issues.
